engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/content-view-available-docker-repositories.controller.js

```diff
@@ -30,6 +30,7 @@ angular.module('Bastion.content-views').controller('ContentViewAvailableDockerRe
 
         var nutupane;
 
+        /*eslint new-cap:0*/
         ContentViewRepositoriesUtil($scope);
```

Comment:
I can try that and see if it works since some of this comes down to how we have defined our services/factories.